assets/app.js

```diff
@@ -668,8 +622,14 @@ new Vue({
                 var response = []
 
                 this.rawMetatags.forEach(function (option, index) {
-                    if(option.name !== '-' && option.is_active==1){
-                        response.push(option);
+                    if(IsAdmin == true){
```

Comment:
same here `DRY` up :slightly_smiling_face:

Reply:
use `this.isAdmin`